src/commands/readme.ts

```diff
@@ -141,13 +145,27 @@ USAGE
 
   renderCommand(config: Config.IConfig, c: Config.Command): string {
     this.debug('rendering command', c.id)
-    const title = template({config, command: c})(c.description || '').trim().split('\n')[0]
-    const help = new Help(config, {stripAnsi: true, maxWidth: columns})
+    const HelpClass = getHelpClass(config)
+    const help = new HelpClass(config, {stripAnsi: true, maxWidth: columns})
+
     const header = () => `## \`${config.bin} ${this.commandUsage(config, c)}\``
```

Comment:
I would love for this to be configurable too via the help class or some other mechanism. That can obviously be done in another PR though.